src/themes/teams/components/Icon/svg/ProcessedIcons/icons-info.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+import { TeamsSvgIconSpec } from '../types'
+
+export default {
+  icon: ({ classes }) => (
+    <svg role="presentation" focusable="false" viewBox="8 8 16 16" className={classes.svg}>
```

Comment:
What should this class do? Should we define it in the styles so that the svg like in other examples?
```
  styles: { 
    svg: ({ variables }) => { 
      return { 
        fill: variables.color, 
      } 
    }, 
  },
```

Reply:
It's a good question. It was already there when I started importing icons, but I don't think we have a use case for it yet/anymore. In the styles, I have this 
```
  svg: ({ props: { size, disabled }, variables: v }): ICSSInJSStyle => {
    return {
      fill: getIconColor(v.color),

      ...(disabled && {
        fill: v.disabledColor,
      }),

      ...getSvgStyle('svg'),
    }
  },
```

So, I think we'd only use it if the icon itself is overriding it's fill color. However, the cases I've seen for overriding would happen on paths, not the entire svg. Do you see a reason for it or should I just remove it from all the svg files?